auth/requirements.txt

```diff
@@ -1,5 +1,6 @@
-aiohttp>=3.3.0,<4.0.0
+aiohttp>=3.7.3,<4.0.0
```

Comment:
Do we have to bump the aiohttp minimum pin? I'd really rather avoid artificially limiting our users if we can help it.